### PR TITLE
Refactor to main

### PR DESCRIPTION
**Pull request by danifitz**
## Summary

Fixes an issue where Terraform registry says this has no root. I refactored all diff files into a single `main.tf`

## How did you test this change?

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ The terraform module will create the following in your AWS account
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.75.2 |
-| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | 0.20.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | ~> 3.0 |
+| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | ~> 0.20.0 |
```
Comment (afiune) [inline README.md]: can we bump this to be `~> 1.0`

**Review by afiune — CHANGES_REQUESTED** [review]
One thing to fix, then it is good to go.

As a side note, I don't think you needed to move "all" the code into the `main.tf`, but instead, Terraform was complaining that the file itself didn't exist. 😅

File: README.md (hunk 1 above)
Comment (danifitz) [reply README.md]: Bumped!

**Review by afiune — APPROVED** [review]
We have three issues that, at some point, we need to resolve. @danifitz I am giving my +1 since this was a copy and paste but, this is why we need Pull Requests so that we notice these kinds of vulnerabilities and other things that run in CI.

Please, make sure to come back to this issues at your earliest convenient. Thanks

tenor-69678117

**Pull request merged**